ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -83,6 +84,18 @@ public static String readShortString(DataInputStream input)
    */
   public static String readIntString(DataInputStream input)
       throws IOException {
+    return readIntString(input, StandardCharsets.UTF_8);
```

Comment:
The old code did "UTF-8" explicitly (it should be part of the diff but might be a bit messed up).